files/es/learn/common_questions/what_are_browser_developer_tools/index.md

```diff
@@ -192,35 +192,35 @@ Esto te dará una ventana como la siguiente:
 
 Para ver qué sucede, intenta ingresar los siguientes fragmentos de código en la consola uno por uno (y luego presiona Intro):
 
-1. ```js
-    alert('hello!');
-    ```
-    
-2. ```js
-    document.querySelector('html').style.backgroundColor = 'purple';
-    ```
-
-3. ```js
-    const myWordmark = document.createElement('img');
-    myWordmark.setAttribute('src','https://blog.mozilla.org/press/wp-content/themes/OneMozilla/img/mozilla-wordmark.png');
-    document.querySelector('h1').appendChild(myWordmark);
-    ```
+```js
+alert('hello!');
+```
+
+```js
+document.querySelector('html').style.backgroundColor = 'purple';
+```
+
+```js
+const myWordmark = document.createElement('img');
+myWordmark.setAttribute('src','https://blog.mozilla.org/press/wp-content/themes/OneMozilla/img/mozilla-wordmark.png');
+document.querySelector('h1').appendChild(myWordmark);
+```
 
 Ahora intenta ingresar las siguientes versiones incorrectas del código y ve lo que obtienes.
 
-1. ```js
-    alert('hello!);
-    ```
-
-2. ```js
-    document.cheeseSelector('html').style.backgroundColor = 'purple';
-    ```
-  
-3. ```js
-    const myWordmark = document.createElement('img');
-    myBanana.setAttribute('src','https://blog.mozilla.org/press/wp-content/themes/OneMozilla/img/mozilla-wordmark.png');
-    document.querySelector('h1').appendChild(myWordmark);
-    ```
+```js
+alert('hello!);
+```
+
+```js
+document.cheeseSelector('html').style.backgroundColor = 'purple';
+```
+
+```js
+const myWordmark = document.createElement('img');
+myBanana.setAttribute('src','https://blog.mozilla.org/press/wp-content/themes/OneMozilla/img/mozilla-wordmark.png');
+document.querySelector('h1').appendChild(myWordmark);
+```
```

Comment:
The en-US version is like this

```suggestion
1. ```js
    alert('hello!);
    ```

2. ```js
    document.cheeseSelector('html').style.backgroundColor = 'purple';
    ```
  
3. ```js
    const myWordmark = document.createElement('img');
    myBanana.setAttribute('src','https://blog.mozilla.org/press/wp-content/themes/OneMozilla/img/mozilla-wordmark.png');
    document.querySelector('h1').appendChild(myWordmark);
    ```
```